test/unit/lib/plugins/aws/package/compile/events/alb/lib/validate.test.js

```diff
@@ -144,6 +144,78 @@ describe('#validate()', () => {
       },
     });
   });
+  it('should accept alb header as both array and key-value object', () => {
```

Comment:
@Inqnuam all new tests need to be configured via `runServerless` util (it's the only reliable way to confirm that functionality is working as expected).

You can find such tests already configured at https://github.com/serverless/serverless/blob/544e25da567de367ec174d491462a57f46a6228c/test/unit/lib/plugins/aws/package/compile/events/alb/index.test.js and I believe you can add one created here as well.

See tests documentation: https://github.com/serverless/serverless/tree/main/test#unit-tests

Reply:
Thanks for the info!
Just added new tests for both single and multiple header conditions.
Please let me know if something is wrong.

Reply:
@Inqnuam if PR is ready for re-review, just re-request review in reviewers box, Thanks 👍